source/_integrations/gandi_livedns.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: Gandi.net
+description: Keep your computer registered with the gandi live dns dynamic updater.
+ha_category:
+  - Network
+ha_iot_class: "Cloud Polling"
+ha_release: 2021.7
+ha_codeowners:
+  - '@flexy2dd'
+ha_domain: gandi_livedns
+---
+
+With the `gandi_livedns` integration you can keep your current IP address in sync with your [Gandi.net](https://www.gandi.net) registered hostname or domain.
+
+<div class='note'>
+Note that it does not create the initial dns record but only updates it; you will still have to do this manually.
+</div>
+
+## Configuration
+
+To use the integration in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+gandi_livedns:
+  domain: domain.com
+  name: hass
+  api_key: YOUR_API_KEY
+  ttl: 3600
+```
```

Comment:
For integrations that connect to devices or services, we no longer accept new YAML configurations or changes.

This integration needs to be refactored first to use a configuration flow and config entries first.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

See our developer documentation on how to get started creating a configuration flow for this integration:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

As these changes often involve a bit of work and some significant shift in the current code, we will close this PR for now.

We (and our community!) would really appreciate it if you could start on the refactoring of this integration in a new PR.

Thanks already! :+1:

Reply:
I have updated integration for support flow and i have updated documentation for reflect this